lib/modules/manager/terraform/lockfile/hash.spec.ts

```diff
@@ -125,4 +140,196 @@ describe('modules/manager/terraform/lockfile/hash', () => {
+      'hashicorp/azurerm',
+      '2.56.0'
+    );
+    expect(log.error.mock.calls).toMatchSnapshot();
+    expect(result).not.toBeNull();
+    expect(result).toBeArrayOfSize(2);
+    expect(result).toMatchObject([
+      'h1:I2F2atKZqKEOYk1tTLe15Llf9rVqxz48ZL1eZB9g8zM=',
+      'h1:I2F2atKZqKEOYk1tTLe15Llf9rVqxz48ZL1eZB9g8zM=',
+    ]);
+  });
+
+  it('contains a ziphash for manifest files not directly used', async () => {
+    const readStreamLinux = createReadStream(
+      'lib/modules/manager/terraform/lockfile/__fixtures__/test.zip'
+    );
+    const readStreamDarwin = createReadStream(
+      'lib/modules/manager/terraform/lockfile/__fixtures__/test.zip'
+    );
+    httpMock
+      .scope(terraformCloudReleaseBackendUrl)
+      .get('/.well-known/terraform.json')
+      .reply(200, terraformCloudSDCJson)
+      .get('/v1/providers/hashicorp/google/versions')
+      .reply(200, terraformCloudBackendGoogleVersions)
+      .get('/v1/providers/hashicorp/google/4.84.0/download/linux/amd64')
+      .reply(200, {
+        os: 'linux',
+        arch: 'amd64',
+        filename: 'terraform-provider-google_4.84.0_linux_amd64.zip',
+        shasums_url:
+          'https://github.com/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_SHA256SUMS',
+        download_url:
+          'https://github.com/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_linux_amd64.zip',
+      })
+      .get('/v1/providers/hashicorp/google/4.84.0/download/darwin/amd64')
+      .reply(200, {
+        os: 'darwin',
+        arch: 'amd64',
+        filename: 'terraform-provider-google_4.84.0_darwin_amd64.zip',
+        shasums_url:
+          'https://github.com/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_SHA256SUMS',
+        download_url:
+          'https://github.com/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_darwin_amd64.zip',
+      });
+
+    httpMock
+      .scope('https://github.com')
+      .get(
+        '/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_SHA256SUMS'
+      )
+      .reply(200, releaseBackendGoogleSha256)
+      .get(
+        '/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_linux_amd64.zip'
+      )
+      .reply(200, readStreamLinux)
+      .get(
+        '/hashicorp/terraform-provider-google/releases/download/v4.84.0/terraform-provider-google_4.84.0_darwin_amd64.zip'
+      )
+      .reply(200, readStreamDarwin);
+
+    const result = await TerraformProviderHash.createHashes(
+      'https://registry.terraform.io',
+      'hashicorp/google',
+      '4.84.0'
+    );
+    expect(log.error.mock.calls).toMatchSnapshot();
+    expect(result).not.toBeNull();
+    expect(result).toBeArrayOfSize(14);
+    expect(result).toContain(
```

Comment:
This does not need a separate test, rather add this to assertion to an other test.

Reply:
Included in the 'full walkthrough on terraform cloud' test, since that one now uses the google provider with manifest.json file.